docs/application/REDUCED_MOTION.md

```diff
@@ -0,0 +1,40 @@
+# Reduced Motion Setup
+The ``@o3r/application`` module provides ``prefersReducedMotion()`` -- a helper that identifies the computer animation preferences.
+This setup is generally used in the scope of accessibility as some users find it difficult to interact with pages with many animations.
+
+To respect their choice, the application's angular animations should follow the computer preferences.
+
+This can be done in the BrowserAnimationsModule's configuration as follows:
+```typescript
+// Other imports
+import {prefersReducedMotion} from '@o3r/application';
+
+@NgModule({
+  imports: [
+    //...
+    BrowserAnimationsModule.withConfig({
+      disableAnimations: prefersReducedMotion()
+    }),
+    //...
+    ]
+})
+export class AppModule {}
+```
+
+**Note:** Any new application that installs the ``@o3r/core`` via the ``ng add @o3r/core`` command will follow this recommendation.
```

Comment:
It is not part of a `ng update` on purpose?

Reply:
As discussed in the original story, the aim is to apply our recommandation on the new applications. I could add an ng-update but is it something we want to apply on new project without forewarning?